src/org/parosproxy/paros/model/Model.java

```diff
@@ -466,8 +466,8 @@ public boolean accept(File dir1, String fileName) {
 		DBNAME_COPY++;
 	}
 
+	@Deprecated
 	public void addSessionListener(SessionListener listener) {

```

Comment:
Remove? From a look at the history it seems this (`sessionListeners`) was never in use.

Reply:
Can do, I was just unsure if any add-ons (official or otherwise) used it.

Reply:
Right, lets keep it then :)